gui/qt/installwizard.py

```diff
@@ -462,27 +483,32 @@ def waiting_dialog(self, task, msg):
         t.join()
 
     @wizard_dialog
-    def choice_dialog(self, title, message, choices, run_next, extra_button=None):
+    def choice_dialog(self, title, message, choices, run_next, extra_button=None,extra_checkbox=None,slp_selection=False):
```

Comment:
This `slp_selection` argument breaks the abstraction here. Why does every choice_dialog need to have this special hack thingy that only possibly applies to the "new" dialog? You're better off making it so the return value includes all relevant information.